telemetry/definitions/commonDefinitions.json

```diff
@@ -1704,6 +1704,11 @@
             "type": "string",
             "description": "A generic version metadata"
         },
+        {
+            "name": "webviewName",
```

Comment:
We already have some existing types such as `name` and `id` which we should resuse. The idea is that these are generalized types that mean different things in the context of the metric that they are in.

Reply:
Didn't know those existed. Apparently you need to search by `: "name"` so it doesn't get clobbered together with the other type names